gomanifest/internal/manifest.go

```diff
@@ -49,19 +49,35 @@ func transformVersion(inVersion string) string {
 
 // getPackageName ... Utility function to convert package + module data into package name used by manifest.
 func getPackageName(depPackage DepPackage) string {
-	// Get module / package@module
 	if depPackage.ImportPath != depPackage.Module.Path {
+		// Get package name like package@module
+		if depPackage.Module.Replace != nil {
+			return depPackage.ImportPath + "@" + depPackage.Module.Replace.Path
+		}
 		return depPackage.ImportPath + "@" + depPackage.Module.Path
```

Comment:
nit: Since there is a little manipulation happening on the ImportPath  and ModulePath, don't repeat it.
```suggestion
                 var importPath, modulePath string
		if depPackage.Module.Replace != nil {
			importPath,  modulePath = depPackage.ImportPath, depPackage.Module.Replace.Path
		} else {
		        importPath,  modulePath = depPackage.ImportPath, depPackage.Module.Path
		}
		return importPath + "@" + modulePath
```

Same for `getPackageVersion`. Just my preference, not necessary to address it.

Reply:
Made code pretty now with similar approach as suggested above. PTAL